src/stcal/jump/jump.py

```diff
@@ -250,6 +282,11 @@ def detect_jumps(frames_per_group, data, gdq, pdq, err,
             previous_row_above_gdq = row_above_gdq.copy()
             k += 1
         elapsed = time.time() - start
```

Comment:
Is this computation of `elapsed` ever actually used or reported anywhere? I see it's repeated again down in line 291 where it is printed to the log.

Reply:
I think it is a relic from the split off of STCAL. The actual time calculation is done in the JWST code now. I'll remove it.

Reply:
No, I was wrong the calculation is here but there was redundant code. I removed it.